server/lib/src/server/mod.rs

```diff
@@ -1630,7 +1665,7 @@ impl<'a> QueryServerWriteTransaction<'a> {
     #[instrument(level = "info", skip_all)]
     pub fn commit(mut self) -> Result<(), OperationError> {
         self.reload()?;
-
+        info!("here : commiting qs_write");
```

Comment:
delete log message, it's already implied by the instrument line.

Reply:
Sorry, all the info!() was to debug stuff in testing....removing them